### PR TITLE
Send page reload message to iOS runtime

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by justcodebuilduser** [comment]
:heart:

File: helpers.ts [hunk 1]
```diff
@@ -157,3 +157,26 @@ export function appendZeroesToVersion(version: string, requiredVersionLength: nu
 
 	return version;
 }
+
+export function whenAny<T>(...futures: IFuture<T>[]): IFuture<IFuture<T>> {
+	let resultFuture = new Future<IFuture<T>>();
+	let futuresLeft = futures.length;
+	let futureLocal: IFuture<T>;
```
Comment (teobugslayer) [inline helpers.ts]: this variable belongs to the for body

**Comment by teobugslayer** [comment]
:+1: after addressing my small remark

**Comment by justcodebuilduser** [comment]
:heart:

**Pull request merged**